### PR TITLE
Update Release 3 notes

### PR DESCRIPTION
**Pull request by osfrickler**
Add some more OSISM related details and cleanup some minor issues.

Signed-off-by: Dr. Jens Harbott <harbott@osism.tech>

### CODE REVIEW

**Review by berendt — APPROVED** [review]

File: Release-Notes/Release3.md [hunk 1]
```diff
@@ -137,6 +153,8 @@ updates for R3 at the end of April.
 
 ## List of known issues & restrictions in R3
 
+* Distributed Virtual Routing (DVR) is not officially supported by us, not tested and not recommended.
```
Comment (fkr) [inline Release-Notes/Release3.md:156]: I would not use the phrase "supported by us" - but instead "supported by OSISM".
I actually had the same thought, when I read that change in the OSISM's changelog and wanted to suggest the same wording there.

File: Release-Notes/Release3.md [hunk 2]
```diff
@@ -72,25 +75,38 @@ Per cluster application credentials.
...
 
 ## Deprecations
 
 Deprecations happen according to our [deprecation policy](https://github.com/SovereignCloudStack/Docs/blob/main/Design-Docs/Release-Policies.md#deprecation).
 
-What ... ?
+* Linux bridge support has been deprecated by the Neutron team and marked as experimental.
+  If you are using it in your deployment, we suggest to migrate to OpenVSwitch instead.
+* Debian dropped hddtemp (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1002484),
+  therefore we will remove the ``hddtemp`` service in the next release, as there is
```
Comment (fkr) [inline Release-Notes/Release3.md:101]: I'd suggest a different wording:

"If Linux bridge is used in deployments, migrating to OpenVSwitch is recommended".

For the hddtemp comment:

"therefore ``hddtemp`` will be removed from the next OSISM release"

**Review by fkr — CHANGES_REQUESTED** [review]
see comments.

**Comment by fkr** [comment]
thanks @osfrickler - really good additions, I made a few small comments regarding some wording.

**Review by fkr — APPROVED** [review]
LGTM!

**Comment by fkr** [comment]
thanks @osfrickler

**Pull request merged**